### ISSUE TITLE
Download Futures Order Book Historical Data

### ISSUE THREAD

**Issue by Animouse31** [LUCIT-Systems-and-Development/unicorn-binance-rest-api#62]
### Is your feature request related to a problem? Please describe.

@oliver-zehentleitner 

Can you add a new feature to download Futures Order Book Historical Data thru Rest API?

Binance is providing it thru a whitelisted account. There are two types of historical data we can download T-Depth and S-Depth.

Here's the explanation about it:
https://www.binance.com/en/support/faq/binance-futures-order-book-historical-data-e707685b4e0f408fa7c1b9ccd8ea7aa9

**Comment by oliver-zehentleitner** [comment]
Yes, I can do that, but it will take a while. If you want do implement it your self in the mean time, a PR is appreciated!
